.github/workflows/release.yml

```diff
@@ -5,8 +5,8 @@ on:
     types: [published]
```

Comment:
I hope we can trigger the nightly path for each merge-to-`master` event, and the official build path for a release. Is it possible to achieve that?

Reply:
Yes, I'm working on this. Seems that time-based nightly release and merge-based nightly release can both work.